tests/unit/components/report/setup.js

```diff
@@ -0,0 +1,34 @@
+
+function _hasPassed(test) {
+  return test.state === 'passed';
+}
+
+module.exports = function(data) {
+  let numberOfPasses = 0,
+      numberOfFailures = 0,
+      duration = 0,
+      suites = data;
+
+  suites.forEach(function(suite) {
+    suite.passes = suite.tests.filter(function(test) {
+      return _hasPassed(test);
```

Comment:
If all you're doing is passing the arguments received from the wrapping function then just use the inner function directly like `suite.tests.filter(_hasPassed)`.